legal-api/src/legal_api/services/authz.py

```diff
@@ -88,3 +90,80 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
     if jwt.validate_roles(roles):
         return True
     return False
+
+
+ALLOWABLE_FILINGS: Final = {
```

Comment:
I wonder if this should be external to the code, say, in a db table, so that it could it be updated without a new deployment ?